src/diffusers/schedulers/scheduling_ddim.py

```diff
@@ -121,7 +148,10 @@ def __init__(
         trained_betas: Optional[np.ndarray] = None,
         clip_sample: bool = True,
         set_alpha_to_one: bool = True,
+        variance_type: str = "fixed",
         steps_offset: int = 0,
+        prediction_type: Literal["epsilon", "sample", "velocity"] = "epsilon",
```

Comment:
Note we currently have a config parameter called `predict_epsilon` that is already used in multiple schedulers:
https://github.com/huggingface/diffusers/blob/ab1f01e63415b63937736299d3a770554c83987e/src/diffusers/schedulers/scheduling_dpmsolver_multistep.py#L131

So we cannot really add this `prediciton_type` here without deprecating the other one and also deprecating arguments like this one: https://github.com/huggingface/diffusers/blob/ab1f01e63415b63937736299d3a770554c83987e/examples/unconditional_image_generation/train_unconditional.py#L197